### DISCUSSION TITLE
Schedule tasks on specific days of the week

### DISCUSSION THREAD

**Discussion by kreativermario** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#970] [Ideas]
Currently I can only schedule tasks for weekdays. Is it possible to schedule a task for a specific day of the week like Monday, Tuesday, etc with a sub-menu with checklists?

<img width="356" alt="image" src="https://github.com/davidwernhart/AlDente-Charge-Limiter/assets/80268327/d6b02bfe-f7e6-47cf-b9c8-8d19e32c0f75">

**Comment by MatthiasKerbl** [comment]
> Currently I can only schedule tasks for weekdays. Is it possible to schedule a task for a specific day of the week like Monday, Tuesday, etc with a sub-menu with checklists?
> 
> <img alt="image" width="356" src="https://user-images.githubusercontent.com/80268327/246399632-d6b02bfe-f7e6-47cf-b9c8-8d19e32c0f75.png">

Hi @kreativermario,

Yes that's possible, however, you need to do a new task for each day. Just select at Repeat "Weekly" and choose at "Start" the day and time of the week you want to launch the task.

Best,
Matthias
